airbyte-integrations/connectors/source-medallia/source_medallia/schemas/feedback.json

```diff
@@ -0,0 +1,118 @@
+{
+   "$schema": "http://json-schema.org/schema#",
+   "properties": {
+     "id": {
```

Comment:
Make sure you follow https://docs.airbyte.com/connector-development/schema-reference